waku/waku_archive/archive.nim

```diff
@@ -87,6 +87,12 @@ proc handleMessage*(
 ) {.async.} =
   self.validator(msg).isOkOr:
     waku_archive_errors.inc(labelValues = [error])
+    trace "failed to insert message",
+      hash_hash = computeMessageHash(pubsubTopic, msg).to0xHex(),
```

Comment:
shouldn't it be `msg_hash` instead of `hash_hash`? especially for consistency with all the other logs in the codebase

same comment for the other logs added here

Reply:
good catch probably a copy-pasta mistake.